src/types/algoliasearch.ts

```diff
@@ -19,11 +19,12 @@ import {
   SearchResponse as SearchResponseV4,
   // no comma, TS is particular about which nodes expose comments
   // eslint-disable-next-line prettier/prettier
-  SearchForFacetValuesResponse as SearchForFacetValuesResponseV4
+  SearchForFacetValuesResponse as SearchForFacetValuesResponseV4,
```

Comment:
```suggestion
  SearchForFacetValuesResponse as SearchForFacetValuesResponseV4
```

Reply:
https://github.com/algolia/instantsearch.js/pull/4581/commits/6040086eb0c56a92540a16e1ba1384cce4068c66

Reply:
the issue doesn't happen when typescript is ran in this project, but with the built .d.ts files